test/patch.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+from unittest import *
+import os.path
+import sys
+
+import hirlite
+
+import redis
+
+
+class PatchConnTest(TestCase):
+    def setUp(self):
+        hirlite.patch_connection()
```

Comment:
If you follow my previous comment about returning a function to undo the monkeypatch, please call it on tearDown and assert the proper Redis classes are back